### DISCUSSION TITLE
Floppy Drive

### DISCUSSION THREAD

**Discussion by MikeCook859** [robhagemans/pcbasic#223] [Q&A]
Hi all, among my pointless ebay purchases has been an external floppy drive.  Can I use the mount command to make this my A: drive for PC-Basic? Attempting to follow the example in the online documentation, I tried "mount a:a:"  It returns a syntax error.  Has anyone done this?  Rob, is this even possible?  Thanks!

EDIT: that strange symbol above is supposed to be"a[colon}a{colon}

**Comment by MikeCook859** [comment]
Well, this isn't a proper answer, but I was digging through the old forum and it turns out mount isn't a command but rather an option if you start pcbasic through the command line, which I will try in just a bit.

**Comment by robhagemans** [comment]
Hi, once your operating system can access the floppy drive you should be able to mount it with the `--mount` option, i.e. to make it your PC-BASIC `A:` drive you need `--mount=a:a:` on Windows or `--mount=a:/dev/fd0` on Linux.

May I ask what online documentation example you're referring to and what made you think `mount` is a BASIC command? It's a common misunderstanding, and although I've updated the documentation multiple times to make this in my view as clear as possible, it keeps recurring. So I'm hoping people can tell me what leads them on the wrong track. Thanks!

**Reply by MikeCook859** [reply]
Hey Buddy!  Sorry it took so long to get back to you, life had to happen for a while.  Yeah, after I posted the question I found the old forum and did a better job looking for an answer.  Whereupon I realized you had addressed this before and you had low-key expressed some frustration with the situation.   When I first found pcbasic, I really did spend some time looking at the command line options and I don't know why I didn't realize how it was supposed to work.  the example I was looking at is in the section "Accessing Your Drives."  You know, I've always been this way.  I just scrolled down and found something and typed it in.  If you can get me and others like me, to slow down and read the directions, my old school teachers would really like to have a word with you.
I really do appreciate your time, Rob.  Steel your nerves, I will have other questions.
